### DISCUSSION TITLE
Version naming (1.9.8)

### DISCUSSION THREAD

**Discussion by ducretje-evooq** [kgateway-dev/kgateway#5916] [Q&A]
Hello,

I'm a bit surprised by the last release:
`latest` tag has been applied on `1.9.8` just after `1.10.7`, I guess it's a mistake and it should be `1.10.8`?

Could someone confirm?

Thanks

**Comment by craigbox** [comment]
Hi @ducretje-evooq 

With the move from Gloo OSS to kgateway, we have moved to a 2.x version number.  Hopefully this won't happen again!
